docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -57,13 +57,17 @@ To fully protect endpoints from malware and other cybersecurity threats when usi
 
 When you receive the prompt to approve loading the system extension:
 
-. Open the *System Preferences* application.
-. Select *Security and Privacy*.
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. In the right pane, scroll down to the Security section. Click **Allow** to allow the Endgame system extension to load.
 +
 --
 image::images/fda/sec-privacy-pane.png[]
 --
-. On the Security and Privacy pane, select the *Privacy* tab.
+. Enter the admin’s username and password and click **Modify Settings** for the changes to take effect.
+
++
+image::images/install-endpoint/allow-system-ext.png[]
```

Comment:
cc @matthewscherer for Endgame Sensors

Reply:
👍 The Endgame changes look good.

Reply:
@Trinity2019 I'm going to move these changes to a separate PR since this PR is meant to cover the outdated [instructions](https://www.elastic.co/guide/en/security/master/deploy-elastic-endpoint.html) for installing Elastic Endpoint manually.